configs/sites/gaea-c5/packages.yaml

```diff
@@ -0,0 +1,241 @@
+  packages:
+    all:
+      compiler:: [intel@2022.0.2]
+      providers:
+        mpi:: [cray-mpich@8.1.16]
+    cray-mpich:
+      externals:
+      - spec: cray-mpich@8.1.16%intel@2022.0.2~wrappers
+        modules:
+        - libfabric
+        - craype-network-ofi
+        - cray-mpich/8.1.16
+    esmf:
+      version: [8.3.0b09]
+      variants: ~xerces ~pnetcdf +pio esmf_os=Linux esmf_comm=mpich3
+    python:
+      buildable: false
+      externals:
+      - spec: python@3.9.12
+        modules: [python/3.9.12]
+    autoconf:
+      externals:
+      - spec: autoconf@2.69
+        prefix: /usr
+    automake:
+      externals:
+      - spec: automake@1.15.1
+        prefix: /usr
+    bash:
+      externals:
+      - spec: bash@4.4.23
+        prefix: /usr
+    binutils:
+      externals:
+      - spec: binutils@2.37.20211103
+        prefix: /usr
+    bison:
+      externals:
+      - spec: bison@3.0.4
+        prefix: /usr
+    # Don't use, it's missing the headers
+    #bzip2:
+    #  externals:
+    #  - spec: bzip2@1.0.6
+    #    prefix: /usr
+    cmake:
+      buildable: false
+      externals:
+      - spec: cmake@3.23.1
+        modules: [cmake/3.23.1]
+    coreutils:
+      externals:
+      - spec: coreutils@8.32
+        prefix: /usr
+    cpio:
+      externals:
+      - spec: cpio@2.12
+        prefix: /usr
+    curl:
+      externals:
+      - spec: curl@7.66.0+gssapi+ldap+nghttp2
+        prefix: /usr
+    diffutils:
+      externals:
+      - spec: diffutils@3.6
+        prefix: /usr
+    dos2unix:
+      externals:
+      - spec: dos2unix@7.4.0
+        prefix: /usr
+    ecflow:
+      buildable: False
+      externals:
+      - spec: ecflow@5.8.4+ui+static_boost
+        prefix: /lustre/f2/scratch/Dom.Heinzeller/spack-stack-c5/ecflow-5.8.4
```

Comment:
Are you sure you want to add an external that's in a personal directory?

Reply:
This is temporary, until we have an official install location. Same for qt.